tt/problem_text.html.tt

```diff
@@ -215,6 +215,34 @@ $('.problem_text div.problem_menu li.langs a').click(function() {
   }
 });
 
+function fill_snippets(data, snippet_list) {
+  for (i = 0; i < snippet_list.length; ++i) {
+    var snippet_data = data[snippet_list[i].getAttribute('snippet')];
+    if (snippet_data === undefined) {
+      snippet_data = '';
+    }
+    snippet_list[i].innerHTML = snippet_data;
+  }
+}
+
+function get_snippets_data(problem_id, request_url) {
+  var snippets = $('#p' + problem_id + ' [snippet]');
+
+  $.ajax({
+    method: "GET",
+    url: request_url,
+    data: { snippet_names : $.map(snippets, function(s) { return s.getAttribute('snippet'); }) },
+    traditional : true,
```

Comment:
1) Remove space before `:`
2) Add a comment (in template, see example at the top of this file)